README.md

```diff
@@ -1,5 +1,8 @@
 # dataretrieval: Download hydrologic data
 
+:warning: USGS data availability and format are changing. Beginning in February 2024 the data obtained from legacy profiles will not include new USGS data or recent updates to existing data. To view the status of changes in data availability and code functionality, visit: https://doi-usgs.github.io/dataRetrieval/articles/Status.html
```

Comment:
Do we want to specify "USGS discrete water quality data..."? This might get a lot of people unnecessarily stressed

Reply:
That sounds fine to me. Changed in readme and warning message.